scripts/upload_processor.py

```diff
@@ -0,0 +1,46 @@
+"""A function to process uploaded samples"""
+
+import json
+import google.auth
+from airtable import Airtable
+from google.cloud import secretmanager
+
+
+def get_table(project_config):
+    """Get Table from Airtable"""
```

Comment:
Don't write comments that only repeat the name, as that doesn't provide any additional value. Instead, think of any information that might be helpful in addition to the name, e.g. what should be contained in `project_config`, are there any invariants / preconditions, what is returned? Also prefer descriptive instead of command-like descriptions, i.e. "Get**s** table from Airtable" and end with punctuation.